src/bots/depositor.py

```diff
@@ -107,13 +107,7 @@ def __init__(
     def execute(self, block: BlockData) -> bool:
         self._check_balance()
 
-        modules_id = get_preferred_to_deposit_modules(self.w3, variables.DEPOSIT_MODULES_WHITELIST)
-
-        if not modules_id:
-            # Read messages in case if no depositable modules for metrics
-            self.message_storage.get_messages_and_actualize(lambda x: True)
-
-        for module_id in modules_id:
+        for module_id in self._get_preferred_to_deposit_modules():
```

Comment:
We shouldn't iterate further, otherwise the point of all new checks is lost

Reply:
Done